dask_sql/physical/rex/core/call.py

```diff
@@ -138,6 +138,9 @@ def reduce(self, *operands, **kwargs):
                 )
             ):
                 operands = tuple(map(as_timelike, operands))
+            if is_cudf_type(operands[0]) and isinstance(operands[1], np.timedelta64):
+                operands = (dd.to_datetime(operands[0], unit="s"), operands[1])
```

Comment:
Curious if this is still needed. I stepped through the tests and didn't need this change as the operands were already in unit `'s` from `to_timestamp`

Reply:
Good point, thanks! I think now the gpuCI failures are just ML failures unrelated to this PR, as the same failures are appearing in https://github.com/dask-contrib/dask-sql/pull/1197 right now too.